utils/hct/hcttestcmds.cmd

```diff
@@ -407,18 +407,18 @@ call :check_file smoke.opt.prn.txt find MODULE-PRINT del
 if %Failed% neq 0 goto :failed
 
 set testname=Smoke test for dxc_batch command line
-call :run dxc_batch.exe -lib-link -multi-thread "%testfiles%\batch_cmds2.txt"
+call :run dxc_batch.exe -lib-link -multi-thread "%testfiles%\Inputs\batch_cmds2.txt"
 if %Failed% neq 0 goto :failed
-call :run dxc_batch.exe -lib-link -multi-thread "%testfiles%\batch_cmds.txt"
+call :run dxc_batch.exe -lib-link -multi-thread "%testfiles%\Inputs\batch_cmds.txt"
 if %Failed% neq 0 goto :failed
-call :run dxc_batch.exe -multi-thread "%testfiles%\batch_cmds.txt"
+call :run dxc_batch.exe -multi-thread "%testfiles%\Inputs\batch_cmds.txt"
 if %Failed% neq 0 goto :failed
 
 set testname=Smoke test for dxl command line
 call :run dxc.exe -T lib_6_x "%testfiles%\lib_entry4.hlsl" -Fo lib_entry4.dxbc
```

Comment:
It's true that there is a copy of this in `tools/clang/test/DXC`, but I don't know why it's not in the `Inputs/` subdirectory. It seems more consistent to put it there. Nothing is using it except this script.

Reply:
the copy of batch_cmds.txt in tools/clang/test/DXC is already removed.